src/tools/rustdoc-gui/tester.js

```diff
@@ -226,7 +226,7 @@ async function main(argv) {
             await Promise.race(tests_queue);
         }
     }
-    if (!no_headless && tests_queue.length > 0) {
+    if (tests_queue.length > 0) {
```

Comment:
Was this check intentionally removed? This seems like a potential behavior change.

Reply:
It is in theory impossible because we run them one by one when we have the `--no-headless` option. But that seems like a redundant check so I just removed it. If there is a remaining test, well, it'll be run here.